mteb/tasks/Classification/tha/ThaiRestaurantReviews.py

```diff
@@ -0,0 +1,49 @@
+from mteb.abstasks.AbsTaskClassification import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class ThaiRestaurantRestaReviews(AbsTaskClassification):
```

Comment:
Can the name indicate the actual dataset name?
`WongnaiReviewsClassification` or something?